modify size cluster documents and performance data (#12461)
pingcap/docs: tidb-cloud/tidb-cloud-performance-reference.md

# TiDB Cloud Performance Reference

This document provides [Sysbench](https://github.com/akopytov/sysbench) performance test results of several TiDB cluster scales, which can be taken as a reference when you [determine the cluster size](/tidb-cloud/size-your-cluster.md).

All the performance data is based on the condition: P95 latency < 105 ms.

> All the test environments are based on TiDB version: v6.1.1.
> 

Reviewer: ```suggestion
```

> The transaction model `Read Only`, `Read Write`, `Write Only` represent the workload of `Read`, `Mixed`, `Write`.

Reviewer: ```suggestion
In this document, the transaction models `Read Only`, `Read Write`, and `Write Only` represent read workloads, mixed workloads, and write workloads. 
```

## 2 vCPU performance

Currently, the 2 vCPU support of TiDB and TiKV is still in beta.

> Test scale:

Reviewer: ```suggestion
Test scales:
```

Test results:

**TiDB (2 vCPU, 8 GiB) \* 1; TiKV (2 vCPU, 8 GiB) \* 3**

    | Transaction model | Threads | QPS      | TPS    | Average latency (ms) | P95 latency (ms) |
    |-------------------|---------|----------|--------|----------------------|------------------|
    | Read Only         | 15      | 3,496.77 | 218.55 | 68.63                | 95.81            |
    | Read Write        | 5       | 1,545.90 | 77.29  | 64.68                | 94.10            |
    | Write Only        | 40      | 4,326.57 | 721.10 | 55.47                | 90.78            |

**TiDB (2 vCPU, 8 GiB) \* 2; TiKV (2 vCPU, 8 GiB) \* 3**

Test results:

    | Transaction model | Threads | QPS      | TPS      | Average latency (ms) | P95 latency (ms) |
    |-------------------|---------|----------|----------|----------------------|------------------|
    | Read Only         | 30      | 7,584.08 | 474.00   | 63.29                | 99.33            |
    | Read Write        | 10      | 2,680.58 | 134.03   | 74.61                | 104.84           |
    | Write Only        | 80      | 7,618.77 | 1,269.79 | 63.00                | 97.55            |

## 4 vCPU performance

> Test scale:

Reviewer: ```suggestion
Test scales:
```
